ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConf.java

```diff
@@ -244,6 +258,24 @@ public class ArcticManagementConf {
           .defaultValue("")
           .withDescription("The password for connecting to the database.");
 
+  public static final ConfigOption<Integer> DB_CONNECT_MAX_TOTAL =
+      ConfigOptions.key("database.connection-pool-max-total")
+          .intType()
+          .defaultValue(20)
+          .withDescription("Max connect count of database connect pool.");
+
+  public static final ConfigOption<Integer> DB_CONNECT_MAX_IDLE =
+      ConfigOptions.key("database.connection-pool-max-idle")
+          .intType()
+          .defaultValue(16)
+          .withDescription("Max idle connect count of database connect pool.");
+
+  public static final ConfigOption<Long> DB_CONNECT_MAX_WAIT_MILLIS =
+      ConfigOptions.key("database.connect-max-wait-millis")
```

Comment:
This is also the connection pool configuration and should use the same prefix `connection-pool-`.

Reply:
Yes, `database.connection-pool-max-wait-millis` is proper here.